graphql/queries/addRoom.graphql

```diff
@@ -0,0 +1,7 @@
+mutation addRoom($name: String!) {
```

Comment:
In the Back-end, it was built with a name parameter. Since we are missing Pond and Thom, I would rather wait before changing how we want to handle data.